torch_geometric/explainability/algo/base.py

```diff
@@ -0,0 +1,168 @@
+from abc import abstractmethod
```

Comment:
This file should also not be in `algo` package.

Reply:
Where should I put it ?

Reply:
I would put it into `torch_geometric/explain/base.py`